core/Cargo.toml

```diff
@@ -108,7 +108,7 @@ strip = "debuginfo"
 [lib]
 name = "comet"
 # "rlib" is for benchmarking with criterion.
-crate_type = ["cdylib", "rlib"]
```

Comment:
drive-by cleanup to fix a warning since we upgraded to Rust 1.79. Unrelated to other changes in this PR.